File: turisticka_agencija_mobile/lib/models/Firma.dart
import 'package:json_annotation/json_annotation.dart';

part 'Firma.g.dart';

@JsonSerializable()
class Firma {
    String adresa;
    String brojZiroracuna;
    int gradId;
    int id;
    String naziv;

    Firma({required this.adresa, required this.brojZiroracuna, required this.gradId,
        required this.id, required this.naziv});

    factory Firma.fromJson(Map<String, dynamic> json) => _$FirmaFromJson(json);

    Map<String, dynamic> toJson() => _$FirmaToJson(this);
}

File: TuristickaAgencija.Model/Firma.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TuristickaAgencija.Model
{
    public class Firma
    {
        public int Id { get; set; }

        public string Naziv { get; set; }

        public int GradId { get; set; }

        public string Adresa { get; set; }

        public string BrojZiroracuna { get; set; }
    }
}
